dsl_parser/functions.py

```diff
@@ -633,6 +633,79 @@ def evaluate(self, handler):
         return results
 
 
+@register(name='get_attributes_dict', func_eval_type=RUNTIME_FUNC)
+class GetAttributesDict(Function):
+
+    def __init__(self, args, **kwargs):
+        self.node_name = None
+        self.attribute_path = None
+        super(GetAttributesDict, self).__init__(args, **kwargs)
+
+    def parse_args(self, args):
+        self.node_name = args[0]
+        self.attribute_paths = args[1:]
```

Comment:
this intrinsic function syntax is very ugly, because it's `{get_attributes_dict: [node, attr1, attr2]}`, so we have a list that is nowhere near homogenous, because it has both the node name, and the attr names. Wouldn't it be better to do `{get_attributes_dict: [node, [attr1, attr2]]}` at least?

Reply:
After a bit of further discussion on slack, we have decided to leave this using the current syntax as it's closer to the other get_attribute functions